src/timeline/media/css/main.css

```diff
@@ -1,174 +1,542 @@
-.track_menu { cursor: pointer; float:left;margin-top:4px;margin-left:4px;margin-right:4px;width:12px;height:12px;background-image: url(../images/menu.png);}
-.track_menu:hover { background-image: url(../images/menu_hover.png); }
-.track_lock { float: left; background-image: url(../images/lock.png); width:14px;height:14px; margin-top: 2px; }
-.track_label { padding-top: 2px; padding-left: 2px; padding-left: 4px; text-shadow: 0 0 10px #ffffff; }
-.track_name { width: 140px; height: 64px; color: #fff; font-size: 9pt; margin-left: 5px; margin-bottom: 8px; background-color: #000; border: 1px solid #4B92AD; border-top-left-radius: 8px; border-bottom-left-radius: 8px; box-shadow: 0px 0px 10px #000; }
-.track_disabled { background: -webkit-gradient(linear, left top, left bottom, color-stop(0%,rgba(20,20,20,1)), color-stop(100%,rgba(6,6,6,1))) !important; }
-.track { height: 64px; background-color: #000; margin-bottom: 8px; background: -webkit-gradient(linear, left top, left bottom, color-stop(0%,rgba(50,50,50,1)), color-stop(100%,rgba(6,6,6,1))); border-top: 1px solid #4b92ad; border-bottom: 1px solid #4b92ad; border-right: 1px solid #4B92AD; border-top-right-radius: 8px; border-bottom-right-radius: 8px; box-shadow: 0px 0px 10px #000; }
+/* Left-column track labels/controls/etc. */
+#track_controls {
+  width: 140px;
+  position: relative;
+  float: left;
+  height: 316px;
+  overflow: hidden;
+}
+
+.track_top {
+  background: -webkit-gradient(linear, left top, left bottom,
+    color-stop(0%, rgba(255, 255, 255, 0.20)),
+    color-stop(100%, rgba(255, 255, 255, 0)));
+  border-top-left-radius: 8px;
+  border-bottom-left-radius: 8px;
+}
+
+.track_lock {
+  float: left;
+  background-image: url(../images/lock.png);
+  width: 14px;
+  height: 14px;
+  margin-top: 2px;
+}
+
+.track_label {
+  padding-top: 2px;
+  padding-left: 2px;
+  padding-left: 4px;
+  text-shadow: 0 0 10px #fff;
+}
+
+.track_name {
+  width: 100%;
+  height: 64px;
+  color: #fff;
+  font-size: 9pt;
+  margin-left: 5px;
+  margin-bottom: 8px;
+  background-color: #000;
+  border: 1px solid #4b92ad;
+  border-top-left-radius: 8px;
+  border-bottom-left-radius: 8px;
+  box-shadow: 0 0 10px #000;
+  box-sizing: border-box;
+}
+
+.track_disabled {
+  background: -webkit-gradient(
+    linear, left top, left bottom,
+    color-stop(0%, rgba(20, 20, 20, 1)),
+    color-stop(100%, rgba(6, 6, 6, 1))) !important;
+}
+
+/* Right-column track timeline container */
+.track {
+  height: 64px;
+  width: 100%;
+  margin-bottom: 8px;
+  background: -webkit-gradient(linear, left top, left bottom,
+    color-stop(0%, rgba(50, 50, 50, 1)),
+    color-stop(100%, rgba(6, 6, 6, 1)));
+  border-top: 1px solid #4b92ad;
+  border-bottom: 1px solid #4b92ad;
+  border-right: 1px solid #4b92ad;
+  border-top-right-radius: 8px;
+  border-bottom-right-radius: 8px;
+  box-shadow: 0 0 10px #000;
+  box-sizing: border-box;
```

Comment:
Ironically, the problem with vertical placement of the clips/transitions was the _lack_ of `border-box` sizing on the track `DIV`s, which weren't the same size we expected them to be due to their borders pushing them out of place. We don't actually _care_ about the size of the tracks themselves, we just care that they're in the positions we expect them to be. So, using `box-sizing: border-box` ensures that the (bordered) track containers will be the same size as the (now-unbordered) draggables, and they line up correctly.